model/registry/process.yaml

```diff
@@ -76,3 +127,22 @@ groups:
           An additional description about the runtime of the process, for example
           a specific vendor customization of the runtime environment.
         examples: 'Eclipse OpenJ9 Eclipse OpenJ9 VM openj9-0.21.0'
+      - id: start
+        type: string
+        brief: >
+          The date and time the process started, in ISO 8601 format.
+        examples: ['2023-11-21T09:25:34.853Z']
+      - id: end
+        type: string
+        brief: >
+          The date and time the process ended, in ISO 8601 format.
+        examples: ['2023-11-21T09:26:12.315Z']
```

Comment:
https://github.com/open-telemetry/semantic-conventions/issues/660

Linking this with this issue, since this will introduce another 2 instances of date time values

Reply:
I'd like to resolve date/time recommendations early here.  I.e. let's not continue digging a hole of inconsistent date/time representation.  

Ideally, I think all date/timestamps should be representing similarly to *raw* timestamps in OTLP.

Reply:
What does raw timestamps mean in that context? Looking into OTLP docs/code I guess unix timestamp in nano seconds (in a 64bit int)?

Reply:
Yeah, I mean unix timestamp in nanoseconds.

I'd like to hold of merging this until we can resovle this dicussion.

Reply:
@jsuereth should we move that conversation over to https://github.com/open-telemetry/semantic-conventions/issues/660 then or is there another issue prior to that where this has been discussed?

Reply:
Sure, #660 is a decent place. We had some in person discussions in the Specification meeting and the Semconv meeting.

I'm not sure we have consensus here, but I *do* think we can likely make progress by sticking (consistently) to String-dates in ISO 8601 format in semconv (what you have).

Reply:
Let's discuss monday to gain consensus.

I think we could take a stand that no matter what the outcome of #660 is, we'll ensure *on the wire* ISO-8601 strings is what people see from OTel semconv.  This would then just be a limitation we place on the solution space for #660.  From what I gathered in both discussions, I think this would be acceptable, but want to run it past the working-group.

Reply:
@jsuereth do we have an outcome that we are staying with ISO-8601 string and this conversation could be resolved? Or you want additional change in this PR regarding time format?

Reply:
Hi @jsuereth, has this been resolved? Should we stay with IS0-8601 in the registry schema?

Reply:
Wow, sorry for not updating this.  

YES sticking with ISO-8601 was the consensus.  Regardless of future work, we think this *is* the representation we want for times as it's the most user-friendly across the most backends.  Any 'advanced' types we may support will need to deal with the decision that, within OTLP, we want ISO-8601 strings.

Thought I had commented on this bug, but apparently just dropped the ball, much apologies!